packages/ui/src/models/camel-entities/EntitySchemaConfig.ts

```diff
@@ -0,0 +1,86 @@
+import { EntityType } from './base-entity.ts';
+import { Schema } from '../schema.ts';
+
+interface IEntitySchema {
+  name: string;
+  schema: Schema | undefined;
+  multipleRoute: boolean;
+}
+
+interface IDslConfig {
+  [EntityType.Route]: IEntitySchema;
+  [EntityType.Kamelet]: IEntitySchema;
+  [EntityType.Pipe]: IEntitySchema;
+  [EntityType.KameletBinding]: IEntitySchema;
+  [EntityType.Integration]: IEntitySchema;
+  [EntityType.Rest]: IEntitySchema;
+  [EntityType.RestConfiguration]: IEntitySchema;
```

Comment:
I think that for DSL purposes, we should only have the ones that affects the Source Code, meaning that in this interface it should be:

* Kamelet
* KameletBinding
* Integration
* Pipe
* Route

Reply:
Thanks. I agree I was just following what we have before.